lib/xgu/xgux.h

```diff
@@ -0,0 +1,88 @@
+#ifndef __XGUX_H
+#define __XGUX_H
+
+#include "xgu.h"
+
+inline
+void xgux_draw_arrays(XguPrimitiveType mode, unsigned int start, unsigned int count) {
+    uint32_t *p = pb_begin();
+    p = xgu_begin(p, mode);
+    p = xgu_draw_arrays(p, start, count);
+    p = xgu_end(p);
+    pb_end(p);
+}
+
+inline
+void xgux_draw_elements16(XguPrimitiveType mode, unsigned int count, const uint16_t* elements) {
```

Comment:
These supposedly have a maximum number of vertices / primitives / ...?
Same issue as for the `pb_begin/end` hack below: might need additional logic.